### PR TITLE
Add autotracker analytics

### PR DESCRIPTION
**Pull request by BenJam**
Adds autotracker.js to libraries.io

BUG: does not concatenate ‘footer’ in layouts/_footer. Fix appreciated! 

TODO: 

- [ ] Add definitions, custom metrics and calculated metrics to GA.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% ga_tracked_els ['footer'] %>
 <footer>
```
Comment (BenJam) [inline app/views/layouts/_footer.html.erb]: it *should* be able to track the el rather than the id. Can do though.

**Comment by BenJam** [comment]
Once this is okay I'll replicate my local analytics setup on Libraries.io's

**Comment by andrew** [comment]
Example:

    <div data-ga-track="dependencies"> 

Which can be selected with jquery using:

    $('[data-ga-track]')

**Comment by BenJam** [comment]
Followed approach from #1216 ¯\_(ツ)_/¯

**Pull request merged**